CHANGELOG.md

```diff
@@ -11,8 +11,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - `tt replicaset promote`: command to promote an instance in the tarantool replicaset with
  cluster config (3.0) or cartridge orchestrator.
+- `tt replicaset demote`: command to demote an instance in the tarantool replicaset with
+ cluster config (3.0) orchestrator.
```

Comment:
By 'config source' I meant etcd or tarantool config storage. Apparently, need to replace it with 'config storage'. 

About `tt rs` for 3.0:
- `promote/demote/etc` patch only local config.yml and does not think about etcd. It is the case when you have a local application and want to manipulate with it. If someone holds the config in etcd, they need to use `tt cluster rs` and be sure that instances config reload it configured properly (with `auto`). `tt cluster rs` will prompt to choose the key for config patching. 

About `tt rs` for cartridge:
- `promote` is supported, `expel`, `join`, I guess, will be supported.
- It can work with local/remote instances (in the most cases) because we can tell the cartridge to patch the config or promote the instance and don't think about how it does this.